package.json

```diff
@@ -24,28 +24,32 @@
     "index.js"
   ],
   "keywords": [
+    "256",
     "ansi",
-    "styles",
+    "cli",
```

Comment:
That's up to @sindresorhus. It appears that, despite my best efforts, package.json got manhandled by NPM. These just got sorted.